spec/editors_draft.html

```diff
@@ -3298,6 +3298,231 @@ <h2>Evaluation Semantics</h2>
+        For example,
+      </p>
+        
+      <pre id="load-renaming-blank-example"
+         data-transform="updateExample"
+         data-content-type="text/turtle"
+         class="nohighlight example">
+      <!--
+      _:b1 ex:p4 "[ <http://example.org/alice>, <http://example.org/bob>, _:eve ]"^^cdt:List .
+      -->
+      </pre>
+      
+      <p>
+        Parsing this data as Turtle will result in a single triple whose subject is a blank node (now dissociated from the `b1` identifier), and whose object is a three-element <a>cdt:List literal</a>.
+        However, using a Turtle parser not aware of Composite Values will result in an object literal containing the substring "_:eve".
+        This presents a challenge, as allowing values to reference specific blank nodes by identifier in this way would allow multiple documents to reference the same blank node, and might expose <em>internal</em> blank node identifiers as <em>external</em> identifiers (a form of unintentional <a href="https://www.w3.org/TR/rdf11-concepts/#dfn-skolem-iri">skolemization</a>).
+      </p>
+        
+      <p>
+        The following sections describe requirements for conforming SPARQL systems that must be applied to composite values during their import into and export from datasets to support proper handling of blank nodes.
+      </p>
+    </section>
+
+    <section>
+      <h2>Importing Requirements</h2>
+      <p>
+        Conforming implementations MUST process <a>cdt:List literal</a>s and <a>cdt:Map literal</a>s during loading, replacing the substrings containing the serialized form of any blank nodes with identifier <var>id</var> with a blank node label <var>b</var>, such that
+      </p>
+      
+      <ul>
+        <li>within the scope of the document being processed, all occurrences of <var>id</var> are replaced with the same <var>b</var> value, and</li>
+        <li>within the scope of the document being processed, two distinct <var>id</var> values are replaced with distinct <var>b</var> values, and</li>
+        <li>
+        	if the document being processed has a mechanism for representing blank nodes outside of composite values (e.g. `_:b1` syntax in N-Triples or Turtle, `rdf:nodeID` in RDF/XML),
+        	and the blank node identifier <var>id</var> appears both inside and outside of composite values,
+        	and the parser emits a blank node <var>B</var> for occurrences of <var>id</var> outside of composite values,
+        	then the rewritten blank node label <var>b</var> MUST be a lexical form that maps to <var>B</var> in the lexical-to-value mapping (that is, <i><a>bnl2bn</a></i>(<var>id</var>) = <var>B</var>).</li>
+      </ul>
+
+      <p>
+        Here, "loading" is any single process by which composite values enter a SPARQL system or RDF Dataset. This includes (but is not limited to):
```

Comment:
The reason I used "single" here was to emphasize that multiple such processes need to be treated independently. For example, `LOAD <test.ttl> ; LOAD <test.ttl>` is a single SPARQL Update request, but it will result in multiple "loading processes" which need to be handled independently (for the same reasons why you'd need to take care to handle blank nodes in the normal [non-cdt] RDF data of test.ttl – it has to result in multiple blank nodes, and that loading cannot be idempotent).

Reply:
Happy to change "single". I'm not sure about "isolated." I considered "every individual, discrete process," but think that might be a bit redundant. My dictionary says of "discrete": "individually separate and distinct," so it covers three of your suggestions (😆). What do you think? I think I'd prefer either my redundant language or just "discrete," but also want this to be clear to non-native speakers, so your opinion is very valuable here.

Reply:
"every individual, discrete process" sounds good.